modules/extension/xsd/xsd-fes/src/main/java/org/geotools/filter/v2_0/bindings/BinarySpatialOpTypeBinding.java

```diff
@@ -55,6 +62,18 @@ public Class getType() {
     
     @Override
     public Object getProperty(Object object, QName name) throws Exception {
-        return FESParseEncodeUtil.getProperty((BinarySpatialOperator) object, name);
+        Expression e = FESParseEncodeUtil.getProperty((BinarySpatialOperator) object, name);
+        if (e instanceof Literal && ((Literal) e).getValue() instanceof Geometry) {
+            return new EncoderDelegate() {
+                @Override
+                public void encode(ContentHandler output) throws Exception {                    
+                    Encoder encoder = new Encoder(new org.geotools.gml3.v3_2.GMLConfiguration());
```

Comment:
We're going to have to pick one. So perhaps the question back at you is, why should GML 3.2.1 not be used here?